src/audio/pipeline/pipeline-stream.c

```diff
@@ -311,17 +313,135 @@ static void pipeline_trigger_xrun(struct pipeline *p, struct comp_dev **host)
 	} while (true);
 }
 
+#if CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL
+static int add_pipeline_cps_consumption(struct comp_dev *current,
```

Comment:
I'd call it `pipeline_add_cps_consumption()`

Reply:
ack, lets keep with the ```pipeline_``` name space. I think we now have most base APIs aligning to this but probably some that still need to be updated.